test/acceptance/features/steps/steps.py

```diff
@@ -267,8 +267,10 @@ def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
     time.sleep(60)
     openshift = Openshift()
     result = openshift.get_deployment_envFrom_info(app_name, context.namespace.name)
-    result | should.be_equal_to("[map[secretRef:map[name:binding-request-1]] map[secretRef:map[name:binding-request-2]]]")\
-        .desc(f'{app_name} deployment should contain secretRef: {sbr_name1} and {sbr_name2}')
+    expected_result_1 = f'[map[secretRef:map[name:{sbr_name1}]] map[secretRef:map[name:{sbr_name2}]]]'
```

Comment:
for better readability, could you name these vars as `expected_result_ocp_45` and `expected_result_ocp_46` respectively?

Reply:
Good idea, thx

Reply:
It's just that the difference is in the `oc` tool itself, not the OCP... so I'd rather name it like `expected_result_oc_45`. And add a comment to mitigate any confusion.